### DISCUSSION TITLE
Different CI trigger conditions

### DISCUSSION THREAD

**Discussion by tai271828** [solvcon/devenv#89] [Ideas]
For now we have many different scripts for package and application build, and it will make us have better maintenance quality by building them periodically.

However, building each of them is very time consumptive. We may not want to trigger it for each pull request. A solution may be that let's build comprehensively daily or weekly only, and trigger the shutil test only for pull request.

**Comment by tai271828** [comment]
For example, this pull request is getting merged https://github.com/solvcon/devenv/pull/88 it means that we may want to monitor if this kind of combination of different packages built by devenv work well.

**Comment by yungyuc** [comment]
Yes, I think building packages on a weekly basis sounds like a good balance.
